### ISSUE TITLE
Save function is broken on flatpak

### ISSUE THREAD

**Issue by likemindedpeople** [huderlem/porymap#524]
tldr: Save and Save All don't work on Flatpak on Linux.

---

I'm on Pop OS 22.04 LTS and have tried installing using the Pop! Shop (PopOS package store) and by downloading from Flathub. Neither Save nor Save All works. 

I'm new to the software so have only messed with map editing and wild encounters. Map editing simply doesn't save, though it triggers the asterisk in the title bar at the top of the window, and those edits appear in the Edit History you can access from the Edit menu.

Editing wild encounters not only does _not_ trigger the asterisk, but may be wiping any other edits present in the Edit History. The wiping behavior may be inconsistent or I may be mistaken, but either way wild encounter edits fail to trigger changes in the Edit History or the title bar asterisk.

**Comment by likemindedpeople** [comment]
that's been what I've been doing, but now that I check, I can't select the repo and choose to Open With, as it doesn't list porymap as a program in that menu. perhaps the program has only read-only access? have there been issues like that before?

**Comment by JakobDev** [comment]
PopOS should give the Flatpak Write Permission to that Directory, when you select it. But that did not work for some reason. You should write a Bug Report. In the meantime, you should use [Flatseal](https://flathub.org/apps/details/com.github.tchx84.Flatseal) to give Porymap write permission for the Host System to fix the Issue.

**Comment by likemindedpeople** [comment]
I was able to get a wild encounter edit to save successfully, but I both gave the program every toggleable permission possible and reopened the project. will edit this comment with more details as they come, but thanks!

**Comment by likemindedpeople** [comment]
yes I did so, it's been working like a charm! the linux version seems to run a lot more smoothly than on windows. thanks again!

---

(also while I'm here: I know you're the flatpak mantainer, but the windows version doesn't appear to support dark mode, or at least I couldn't get it to work. cheers!)

**Comment by garakmon** [comment]
Should be
